src/components/text-style-button.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export interface textButtonInterface {
+    text: string
+    onClicked: () => void
+}
+
+
+const ButtonStyle = styled.div`
+    color: #BDBDBD;
+    font-family: Montserrat, Arial;
+    font-size: 14px;
+    cursor: pointer;
+    &:hover {
+        text-decoration: underline;
+    }
+`
+
+function TextButton(props: textButtonInterface) {
+    return <ButtonStyle><p onClick={ props.onClicked }>{ props.text }</p></ButtonStyle>
+}
+
+export default TextButton
```

Comment:
Let's add a newline at the bottom of the file. If you want, you can also setup your editor to automatically add a newline at the end of files and automatically trim newlines.

Reply:
Just saw this one - I'll add the newline in another commit :)